package.json

```diff
@@ -21,10 +21,11 @@
 	"scripts": {
 		"build": "babel --out-dir=dist source",
 		"dev": "babel --out-dir=dist --watch source",
-		"clean": "rm -rf",
-		"setup": "npm run setup:ubuntu",
-		"setup:ubuntu": "npm run clean && npm run build && npm i -g .",
-		"setup:windows":"rmdir /s /q dist && yarn build",
+        "clean:ubuntu": "rm -rf dist",
+        "clean:windows": "rmdir /s /q dist",
+		"setup": "npm run setup:ubuntu || npm run setup:windows",
```

Comment:
I'm not sure if this `||` will work in both systems. Would you happen to have a demo? If we're not sure, let's use the Linux command as the default instead.

Reply:
yes i had tried it on powershell and it worked but it wont work on cmd.exe so we can shift to linux command as default.